lib/util/img-maker.coffee

```diff
@@ -60,12 +43,10 @@ getImgMakerHelper = (imageMakerUrl, request) ->
 		else if maxAge is null
 			maxAge = undefined
 
-		memoizedFn = promiseMemoize(callHelper, { resolve: normalizer, maxAge })
+		memoizedFunction = promiseMemoize(callHelper, { maxAge })
```

Comment:
So this will always stringify args for the key and will use `latest` as the version for the requests (like `/versions`) that don't have this param. Sounds OK, just verifying it.